docs/content/concepts/cryptography/nautilus.mdx

```diff
@@ -57,7 +57,7 @@ When used together, Nautilus and [Seal](https://github.com/MystenLabs/seal) enab
 
 ## Future plans and non-goals
 
-Nautilus will support additional TEE providers in the future, such as [Intel TDX](https://www.intel.com/content/www/us/en/developer/tools/trust-domain-extensions/overview.html) and [AMD SEV](https://www.amd.com/en/developer/sev.html) possibly. Your suggestions on which platforms to prioritize or support is greatly appreciated.
+Nautilus will support additional TEE providers in the future. Your suggestions on which platforms to prioritize or support is greatly appreciated.
```

Comment:
Would be nice to include a link here. I have a suggestion, where do I provide it?

Reply:
Sure, added. Thank you.